jekyll/_cci2/runner-installation.adoc

```diff
@@ -0,0 +1,154 @@
+= Installing the CircleCI Runner
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+This page describes the steps required to install CircleCI Runner on Linux and macOS platforms.
+
+=== Supported Platforms
+
+At present, the supported platforms are:
+
+- Ubuntu 18.04 running on an x86_64 (AMD64) or ARM64 architecture
+- macOS Catalina (10.15.6+) or above, running on an x86_64 architecture.
+
+Running in a container is not supported at this time.
+
+=== Prerequisites
+
+==== Authentication
+
+In order to complete this process you'll need to create a namespace, resource-class and authentication token:
+
+. Install the circleci command-line tool.
+
+. Create a namespace for your organization's runner resources under:
+
+Each organization can only create a single namespace.
+
+- If you already use orbs, this will be same namespace as the orbs use. Use the following command: `circleci namespace create <name> <vcs-type> <org-name>` (e.g. if your GitHub URL is `https://github.com/circleci`, then use: `circleci namespace create my-namespace github circleci`.
+
+. Create a resource class for your runner in the above namespace:
+
+- Use the following command: `circleci runner resource-class create <resource-class> <description>` (e.g. `circleci runner resource-class create my-namespace/my-resource-class my-description`)
+
+. Create a token for authenticating the above resource-class:
+
+- Use the following command: `circleci runner token create <resource-class> <nickname>` (e.g. `circleci runner token create my-namespace/my-resource-class my-token`).
+
+This will print a generated Runner config including the authentication token. Note that the token cannot be retrieved again, so store it safely.
+
+==== Installation Tooling
+
+The installation process assumes that the following utilities are installed on the system:
+
+- curl (installed by default on macOS)
+- sha256sum (installed as part of coreutils on Linux apt/yum, macOS via brew)
+- systemd version 235+ (Linux only)
+- Installation also requires permissions to create a user, and create directories under /opt.
+
+=== Job Running Requirements
+
+Running jobs requires that the following tools are available on the machine:
+
+- tar
+- gzip
+- coreutils (Linux only)
+- git (recommended, but not required)
+
+=== Installation
+
+==== Download the Launch Agent Binary and Verify the Checksum
+
+The Launch Agent can be installed using the following script in a bash shell.
+
+This will use /opt/circleci as the base install location.
+
+First, set one of these variables as appropriate for for your installation target.
+
+===== For Linux x86_64
+
+`platform=linux/amd64`
+
+===== For Linux ARM64
+
+`platform=linux/arm64`
+
+===== For macOS x86_64
+
+`platform=darwin/amd64`
```

Comment:
I would maybe put this in a table instead of under headings?

Reply:
Added a table. Thanks for the suggestion. This is so much better :)